webpack/index.js

```diff
@@ -6,7 +6,7 @@
 import componentRegistry from 'foremanReact/components/componentRegistry';
 import { mount } from 'foremanReact/common/MountingService';
 import ExperimentalUi from './containers/Application/index';
-import './reducers';
+import './redux/reducers';
```

Comment:
👍  to changing this to be consistent with foreman.

Reply:
Hey -- So are we talking an icon that represents repos, or one that represents files in repos?